src/databaseTests/src/test/kotlin/org/vaccineimpact/api/databaseTests/tests/touchstoneRepository/GetScenarioTests.kt

```diff
@@ -213,6 +215,248 @@ class GetScenarioTests : TouchstoneRepositoryTests()
         }
     }
 
+    @Test
```

Comment:
Wow, so this class is now massive and to be honest I'm finding it hard to review, so many similarly named tests!
Could we split into sub classes? Maybe leave the tests that don't actually get coverage data, like the first 4 tests in this class and create two more, `GetCoverageTests` and `GetAggregatedCoverageTests` which test the single row/grouped row logic respectively? Or any other division that makes sense to you, but I'd say this is at least 2 if not 3 classes worth of logic!